packages/bot-web-ui/src/assets/sass/run-panel.scss

```diff
@@ -9,8 +9,7 @@
 
     &__footer {
         text-align: center;
-        height: 152px;
-        margin: 16px 0;
+        margin-top: 16px;
```

Comment:
This glitches the run panel, you can remove this and add a `padding-top: 16px;` to the `.dc-drawer__footer` element instead.

![Screenshot 2020-03-10 at 11 37 20](https://user-images.githubusercontent.com/44427668/76277540-fb26f300-62c3-11ea-9939-c908b8e5a380.png)